man/sd_bus_send.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version='1.0'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1+ -->
+
+<refentry id="sd_bus_send"
+          xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <refentryinfo>
+    <title>sd_bus_send</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>sd_bus_send</refentrytitle>
+    <manvolnum>3</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>sd_bus_send</refname>
+
+    <refpurpose>Queue a D-Bus message for transfer</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <funcsynopsis>
+      <funcsynopsisinfo>#include &lt;systemd/sd-bus.h&gt;</funcsynopsisinfo>
+
+      <funcprototype>
+        <funcdef>int <function>sd_bus_send</function></funcdef>
+        <paramdef>sd_bus *<parameter>bus</parameter></paramdef>
+        <paramdef>sd_bus_message *<parameter>m</parameter></paramdef>
+        <paramdef>uint64_t *<parameter>cookie</parameter></paramdef>
+      </funcprototype>
+    </funcsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><function>sd_bus_send()</function> queues the bus message object <parameter>m</parameter>
+    for transfer. If <parameter>bus</parameter> is <constant>NULL</constant>, the bus that
+    <parameter>m</parameter> is attached to is used. <parameter>bus</parameter> only needs to be set
+    when the message is sent to a different bus than the one it's attached to, for example when
+    forwarding messages. The message is sent to the destination set using
+    <citerefentry><refentrytitle>sd_bus_message_set_destination</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    or its direct peer if <parameter>bus</parameter> is a direct connection.</para>
```

Comment:
I don't like this phrasing, because in many cases sd_bus_message_set_destination() is not called. For replies the destination is usually set implicitly to the sender, e.g.  `sd_bus_message_new_method_error` and anything else that uses `message_new_reply` internally does that. So I think this should be worded more generally.

Reply:
What happens when a destination is set but the connection is a direct connection? Or the reverse, no destination is set but we're connected to a bus?

Reply:
I'm wondering if we should mention the destination at all in this man page. After thinking about it a bit more, the whole destination discussion feels a little out of place here. Perhaps its better to discuss each possibility (unicast, broadcast, direct connection, ...) in the man page of the relevant functions.